src/stories/Library/link-with-icon/link-with-icon.scss

```diff
@@ -0,0 +1,60 @@
+.link-with-icon {
+  background-color: $c-global-primary;
+  padding: $s-md $s-lg;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  border: 1px solid $c-global-tertiary-1;
+  line-height: 160%;
+  width: 100%;
+  @include container(medium);
+
+  > svg {
+    display: block;
+    width: 35px;
+    margin-left: auto;
+    visibility: hidden;
+  }
+
+  &:first-child {
+    margin-top: 0;
+  }
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  & + .link-with-icon {
+    margin-top: 0;
+  }
+
+  @include breakpoint-s {
+    padding: $s-lg $s-xl;
+    &:hover > svg {
+      visibility: visible;
+    }
+  }
+
+  &.arrow__hover--right-small {
+    .arrow__body,
+    .arrow__head {
+      transition: none;
+    }
```

Comment:
Please reconsider this CSS.

When using BEM we do not want our blocks to interfere with eachother. *If* we want to disable animation for an arrow we should instead create a new modifier for `arrow` (e.g. `arrow--no-animation`) , disable transitions for that. Then we can use this modifier in our template for links with icons.

Also: Is ther a reason why we want to disable the default arrow animation here? I cannot tell from code or commit comments.